pkg/find/vulnerabilities/trivy/converter.go

```diff
@@ -79,15 +79,22 @@ func (c *converter) convert(reports []ScanReport) starboard.VulnerabilityReport
 		}
 	}
 
+	registry, artifact, err := c.parseImageRef(imageRef)
+	if err != nil {
+		return starboard.VulnerabilityReport{}, nil
```

Comment:
Do we return `nil`? Even if so, we might need kind of logging.

Reply:
Good catch. I meant to return the error. It's fixed now with a test case to assert that the error is indeed returned